src/cmd/cmdHandler.ts

```diff
@@ -165,4 +180,53 @@ export default class CmdHandler {
 
     return proc;
   };
+
+  _askImageName = async (filepath: string): Promise<string | undefined> => {
+    let fileName = path.basename(filepath);
+    fileName = fileName.replace(path.extname(fileName), "");
+    return vscode.window.showInputBox({
+      value: fileName,
+      placeHolder: "Image name to use"
+    });
+  };
+
+  _askMounts = async (): Promise<string | undefined> => {
+    let volumeIDs = this.ops.listVolumeID();
+    let ids = await vscode.window.showQuickPick(volumeIDs, {
+      placeHolder: "Select volumes to mount",
+      canPickMany: true
+    });
+
+    let mounts: string[] = [];
+    if (ids?.length) {
+      let mountPath: string | undefined;
+      for (let i = 0; i < ids.length; i++) {
+        mountPath = await vscode.window.showInputBox({
+          prompt: `Mount point for volume ${ids[i]}`,
+          placeHolder: "Path to mount the volume"
+        });
+        if (mountPath) {
+          mounts.push(`${ids[i]}:${mountPath}`);
+        }
+      }
+    }
+    console.log(mounts.join());
```

Comment:
cruft?

Reply:
Sorry forgot about that, should be removed by https://github.com/nanovms/ops-vscode/pull/13/commits/fbb5c3a3f3810c7ac8372a96b6989981a0b4a0bf